docs/versioned_docs/version-zenith/guides/snippets/github-google-cloud/python/main.py

```diff
@@ -0,0 +1,61 @@
+import dagger
+from dagger import dag, object_type, field, function
+
+@object_type
+class MyModule:
+
+    @function
+    def build(source: dagger.Directory) -> dagger.Container:
```

Comment:
```suggestion
    def build(self, source: dagger.Directory) -> dagger.Container:
```

Reply:
Could have `source` as a field instead (constructor). This way all methods have access to it.

Reply:
Done